File: API/DTOs/Report/StorageSumReport/Report_Storage_Sum_Param.cs
namespace API.Helpers
{
    public class Report_Storage_Sum_Param
    {
        public string crday { get; set; }
        public string sdat { get; set; }
        public string mdat { get; set; }
        public string brandname { get; set; }
        public string cusna { get; set; }
        public string manno { get; set; }
        public string purno { get; set; }
        public string rmodel { get; set; }
        public string tolcls { get; set; }
        public string bitnbr { get; set; }
        public string kind { get; set; }
        public string startDate { get; set; }
        public string endDate { get; set; }
        public string size { get; set; }
    }

    public class SearchSortSumReportParams
    {
        public string crday { get; set; }
        public string dateStart { get; set; }
        public string dateEnd { get; set; }
        public string brandname { get; set; }
        public string cusna { get; set; }
        public string manno { get; set; }
        public string tolcls { get; set; }
        public string rmodel { get; set; }
        public string purno { get; set; }
        public string bitnbr { get; set; }
        public string kind { get; set; }
        public string etaFrom { get; set; }
        public string etaTo { get; set; }
        public string size { get; set; }
    }
}

File: SPA/src/app/_core/models/Report/report_Storage_Sum_Param.ts
export interface Report_Storage_Sum_Param {
  crday: string;
  sdat: string;
  mdat: string;
  brandname: string;
  cusna: string;
  manno: string;
  purno: string;
  rmodel: string;
  tolcls: string;
  bitnbr: string;
  kind: string;
  startDate: string;
  endDate: string;
  size: string;
}

export interface Report_Storage_Sum {
  crday: string | null;
  mdat: string;
  brandname: string;
  cusid: string;
  cusna: string;
  manno: string;
  purno: string;
  rmodel: string;
  tolcls: string;
  ritnbr: string;
  bitnbr: string;
  article: string;
  kind: string;
  eta: string | null;
  size: string;
  qty: number | null;
  wkshqty: number | null;
  pqty: number | null;
  sort_qty: number;
  storage_qty: number;
  diff_qty: number | null;
  cqty: number | null;
  check: boolean
}
